fda.playground/Contents.swift

```diff
@@ -0,0 +1,3 @@
+import Cocoa
+
+var greeting = "Hello, playground"
```

Comment:
mas tu navic pushnuty nejaky playground... Klidne si ho tu nech, jestli ho budes chtit vyuzivat i v budoucnu. 👍  

Pisu ti to sem spis jenom jako info, jestli si ho tu treba jenom nezapomel a omylem nepushnul. 😉